schema/transactions.js

```diff
@@ -16,17 +22,7 @@ module.exports = {
 			type: {
 				type: 'integer',
 				minimum: 0,
-				maximum: 10
-			},
-			senderId: {
-				type: 'string',
-				format: 'address',
-				minLength: 1,
-				maxLength: 22
-			},
-			senderPublicKey: {
-				type: 'string',
-				format: 'publicKey'
+				maximum: 7
```

Comment:
I am not sure if hardcoding this is correct, since more types should be supported in the future. Perhaps make it a function of the fees array in constants.

Reply:
Yeah, I'll do it. Previously it's value was 10. Which doesn't make sense at all for me.